code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -4,7 +4,7 @@
 #define MOVING_TO_TARGET 3
 #define SPINNING_COCOON 4
 
-/mob/living/simple_animal/hostile/poison
+/mob/living/simple_animal/hostile/poison //this guy's a fukkin idiot everyone knows that spiders inject venom not poison

```

Comment:
It's a subtype of hostile mobs that transfer reagents and aren't exclusive to spiders.

See: Bees.

Reply:
hahahaha bees still inject venom haHA